runway/blueprints/staticsite/staticsite.py

```diff
@@ -107,79 +67,124 @@ class StaticSite(Blueprint):  # pylint: disable=too-few-public-methods
                                                         'associations.'},
     }
 
+    @property
+    def aliases_specified(self):
+        # type: () -> bool
+        """Aliases are specified conditional."""
+        return self.get_variables()['Aliases'] != ['']
+
+    @property
+    def cf_enabled(self):
+        # type: () -> bool
+        """CloudFront enabled conditional."""
+        return not self.get_variables()['DisableCloudFront']
+
+    @property
+    def acm_certificate_specified(self):
+        # type: () -> bool
+        """ACM Certification specified conditional."""
+        return self.get_variables()['AcmCertificateArn'] != ''
+
+    @property
+    def cf_logging_enabled(self):
+        # type: () -> bool
+        """CloudFront Logging specified conditional."""
+        return self.get_variables()['LogBucketName'] != ''
+
+    @property
+    def directory_index_specified(self):
+        # type: () -> bool
+        """Directory Index specified conditional."""
+        return self.get_variables()['RewriteDirectoryIndex'] != ''
+
+    @property
+    def waf_name_specified(self):
+        # type: () -> bool
+        """WAF name specified conditional."""
+        return self.get_variables()['WAFWebACL'] != ''
+
+    def add_parameters(self):
```

Comment:
I actually was considering adding more to this to be honest, isn't it best practice to try and keep that info available after stack creation for review? If we specify on the VARIABLES piece how does that work exactly? The only way I was able to get a parameter to show is if I supplied it a built-in type such as `CFNString` or the like, would we have to double up all our variables then at that point?

Reply:
> ...keep that info available after stack creation for review
The intent here is very good. Deployed CFN stacks without any parameters look somewhat weird.

The problem is that CFN parameters imply their use in the template. E.g. if there's a `PriceClass` parameter, someone interacting with that stack would expect that a change to the parameter would update the CF distribution priceclass -- but for this template, because the priceclass is baked into when the template is made.

As you noted, to make them CloudFormation parameters, any place they were used in conditional logic would need to be replaced with native CloudFormation conditions (which would actually be my preference)

Reply:
Alrighty I'll go ahead and dump the parameters piece then since we decided we weren't going to do the CloudFormation conditions. Thanks!